SoftLayer/managers/ordering.py

```diff
@@ -745,3 +746,14 @@ def delete_quote(self, quote_id):
         """
 
         return self.client['SoftLayer_Billing_Order_Quote'].deleteQuote(id=quote_id)
+
+    def get_all_cancelation(self):
+        """returns the all cancelations, completed orders"""
+
+        mask = 'mask[id,itemCount,modifyDate,createDate,ticketId,' \
```

Comment:
Have the mask be only the columns we need to display, which will make the API command much faster.